runtime/src/main/java/org/corfudb/runtime/exceptions/AbortCause.java

```diff
@@ -10,6 +10,7 @@ public enum AbortCause {
     //TODO(Maithem): these types are redundant and should be removed,
     // we should just expose the throwable
     CONFLICT,
+    DEADLOCK,
```

Comment:
hi George, just wondering where this variable is used?

Reply:
This will be used in datastore as a new txn abort cause.